docs/ValidationRules.md

```diff
@@ -0,0 +1,642 @@
+# Rules
```

Comment:
>Rules [](start = 2, length = 5)

Call this file BinSkimRules.md or simply RuleMessages.md

Reply:
we could consider injecting the tool name, so 'BinSkim Rules'. BinSkimRules.md (with the injected tool name) would be a good default file name.

---
In reply to: [524812423](https://github.com/microsoft/binskim/pull/308#discussion_r524812423) [](ancestors = 524812423)

Reply:
just updated to BinskimRules.md